finality-aleph/src/testing/mocks/validator_network.rs

```diff
@@ -84,6 +93,18 @@ pub fn random_address() -> MockAddressingInformation {
             .map(char::from)
             .take(43)
             .collect(),
+        true,
+    )
+}
+
+pub fn random_invalid_address() -> MockAddressingInformation {
```

Comment:
Why do you want to have randomness in your unit tests?

Reply:
It used to work that way and I did not change this – I think it's alright here, since this specific kind of randomness shouldn't influence _anything_ about code execution, and if it does something is so wrong we want to know about this.